src/notation/view/selectionfiltermodel.cpp

```diff
@@ -196,7 +196,7 @@ QString SelectionFilterModel::titleForType(SelectionFilterType type) const
     case SelectionFilterType::ARPEGGIO:
         return qtrc("notation", "Arpeggios");
     case SelectionFilterType::GLISSANDO:
-        return qtrc("notation", "Glissandi");
+        return qtrc("notation", "Glissandos");
```

Comment:
@oktophonie

Reply:
I seem to recall this being discussed in the past, and considered on the basis that we use e.g. "arpeggios" and "tremolos", so it's more consistent to stick to using English plurals in English. I'm fine with it.

Reply:
I guess that marks this conversation resolved